hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/config/EvmConfiguration.java

```diff
@@ -164,6 +165,18 @@ CacheManager cacheManagerRecordFileIndex() {
         return caffeineCacheManager;
     }
 
+    @Bean(CACHE_MANAGER_RECORD_FILE_TIMESTAMP)
+    CacheManager cacheManagerRecordFileTimestamp() {
+        final var caffeine = Caffeine.newBuilder()
+                .expireAfterWrite(10, TimeUnit.MINUTES)
```

Comment:
```suggestion
                .expireAfterAccess(10, TimeUnit.MINUTES)
```

perhaps also follow the pattern to make it a configurable string with a default value like other cache managers

Reply:
The expiration time is also hardcoded in all other cache manager beans here.